how-does-dtp-work.md

```diff
@@ -7,7 +7,7 @@ DTP comprises three main components:
 Data Models are the canonical formats that establish a common understanding of how to transfer data.
 Adapters provide a method for converting each Provider’s proprietary data and authentication formats into a form that is usable by the system.
 Task Management Library provides the plumbing to power the system.
-Data Models
```

Comment:
Actually I think the right fix for this is to make it a header

### Data Models 

And the above should be a list (adapters and task management have a section below on lines 21 and 31)

Reply:
Thank you @seehamrun for the review